rasa/model_training.py

```diff
@@ -75,6 +79,22 @@ def _dry_run_result(
     return TrainingResult(dry_run_results=fingerprint_results)
 
 
+def _check_unresolved_slots(domain: Domain, stories: StoryGraph) -> None:
+    unresolved_slots = set(
+        evnt.key
+        for step in stories.story_steps
+        for evnt in step.events
+        if isinstance(evnt, SlotSet)
+    ) - set(slot.name for slot in domain.slots)
+    if unresolved_slots:
+        rasa.shared.utils.cli.print_error_and_exit(
+            f"Unresolved slots found in stories/rules🚨 \n"
+            f"Tried to set non existent slots - {', '.join(unresolved_slots)}.\n"
+            f"Make sure you added all your slots to your domain file."
```

Comment:
> "Make sure you added all your slots to your domain file."

I'd suggest amending this wording to:

> "Make sure you add all your slots to the domain file correctly."

As well as reminding users to add *all* of their slots to the domain file, adding "correctly" reminds users to check the spelling of them - a likely cause of this error.

Reply:
Thank you for your suggestions. Rephrased it accordingly